stacks/aws.py

```diff
@@ -0,0 +1,57 @@
+from .utils import throttling_retry
```

Comment:
> aws_helpers or something might be a better name

Probably. Let's worry about that later. I think once we have more functions, etc we will know how to group them better.